Tasks/DotNetCoreInstallerV0/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -6,7 +6,7 @@
   "loc.input.label.packageType": "Package to install",
   "loc.input.help.packageType": "Please select whether to install only runtime or full SDK.",
   "loc.input.label.version": "Version",
-  "loc.input.help.version": "Specify exact version of .NET Core SDK or runtime to install.<br/><br/>Examples:<br/>1. To install 1.0.4 SDK, use 1.0.4<br/>2. To install 1.1.2 runtime, use 1.1.2<br/>2. To install 2.0 preview 2 runtime, use 2.0.0-preview2-25407-01<br/><br/>For getting more details about exact version of runtime (, refer [here](https://github.com/dotnet/core/blob/master/release-notes/releases.json)",
+  "loc.input.help.version": "Specify exact version of .NET Core SDK or runtime to install.<br/><br/>Examples:<br/>1. To install 1.0.4 SDK, use 1.0.4<br/>2. To install 1.1.2 runtime, use 1.1.2<br/>2. To install 2.0 preview 2 runtime, use 2.0.0-preview2-25407-01<br/><br/>For getting more details about exact version refer [here](https://github.com/dotnet/core/blob/master/release-notes/releases.json)",
```

Comment:
The help text is not up-to the mark. Can you specify the exact strings to look for in the json